spec/features/merchants/discounts/new_spec.rb

```diff
@@ -41,16 +41,22 @@
     expect(current_path).to eq("/merchant/#{@bike_shop.id}/discounts/new")
   end
 
-  it "can creat a new discount" do
+  it "can create a new discount and assign it to an item" do
     click_link "New Discount"
 
     fill_in :name, with: "30% Discount"
     fill_in :percent_off, with: 30
     fill_in :min_quantity, with: 100
+    within "#item-#{@chain.id}" do
+      check :discount_items_
+    end
 
     click_button "Create Discount"
 
     discount = Discount.last
+    item_disount = ItemDiscount.last
+    expect(item_disount.discount).to eq(discount)
+    expect(item_disount.item).to eq(@chain)
 
```

Comment:
Add a test that checking an item upon discount creation puts that item and discount in the joins table